spec/model/post_spec.rb

```diff
@@ -0,0 +1,31 @@
+require 'rails_helper'
+
+RSpec.describe Post, type: :model do
+  let(:user) { User.new(name: 'John', photo: 'https://www.google.com', bio: 'I am a user') }
+  subject { Post.new(title: 'Title', text: 'Text', author: user) }
+  before { subject.save }
+
+  it 'has title' do
+    expect(subject.title).to eq('Title')
+  end
+
+  it 'has text' do
+    expect(subject.text).to eq('Text')
+  end
+
+  it 'title has a maximum length of 250 characters' do
+    expect(subject.title.length).to be <= 250
+  end
+
+  it 'likes_counter must be 0 or greater' do
+    expect(subject.likes_counter).to be >= 0
+  end
+
```

Comment:
- [ ] @heintayzar-hm great work on your tests kindly add a test for the comment counter validation.

Reply:
- [ ] @heintayzar-hm it would be nice if you added tests that checks for title validation eg a post can be created if it doesn't have title or tile is longer than 250 characters